css/contact.css

```diff
@@ -35,16 +36,16 @@ textarea {
 	background-color: var(--color-white);
 	border: 1px solid #6c6c6c;
```

Comment:
is it possible to have border color a var? feel free to add it to **global.css**

Reply:
added variable to global.css as --color-grey and replaced in contact.css

Reply:
Yay!